src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: New Relic OTLP Endpoint
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - OTLP
+metaDescription: Configure OTLP exporter for New Relic
+freshnessValidatedDate: never
+---
+
+[OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-proto/blob/main/docs/specification.md), or OTLP for short, is a general purpose telemetry data delivery protocol designed for the OpenTelemetry project. Each OpenTelemetry language SDK provides OTLP exporters, and the OpenTelemetry collector has OTLP receivers and exporters. Additionally, various tools outside the OpenTelemetry project have added support for OTLP export.
+
+New Relic supports native OTLP ingest, and recommends OTLP as the means for sending OpenTelemetry data to New Relic. This document describes New Relic OTLP support, including configuration requirements and recommendations.
```

Comment:
```suggestion
New Relic supports native OTLP ingest and recommends it as the preferred method for sending OpenTelemetry data to the New Relic platform. This document delves into New Relic's OTLP support, including configuration requirements and best practices.
```

Reply:
I'd like to keep recommendations. Later in the doc, I characterize configuration sections as required or recommended. This phrasing here is a reference to that. Recommended is a better description of these options than calling them a best practice. 

Otherwise, I agree.